configure

```diff
@@ -7249,22 +7249,22 @@ fi
 test -z "$MANIFEST_TOOL" && MANIFEST_TOOL=mt
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if $MANIFEST_TOOL is a manifest tool" >&5
 printf %s "checking if $MANIFEST_TOOL is a manifest tool... " >&6; }
-if test ${lt_cv_path_mainfest_tool+y}
+if test ${lt_cv_path_manifest_tool+y}
```

Comment:
As noted, I can't find this in an upstream (there are [instances around](https://cs.github.com/?scopeName=All+repos&scope=&q=lt_cv_path_mainfest_tool), but none are obvious upstreams, which hints that it was fixed upstream at some point).

An alternative would of course be to delete these files. The README hints that autoconf is deprecated...